crawler/file_processing.py

```diff
@@ -783,25 +778,27 @@ def check_for_required_headers(self, csvreader: DictReader) -> bool:
         return True
 
     def extract_plate_barcode_and_coordinate(
-        self, row: Dict[str, Any], line_number, barcode_field: str, regex: str
+        self, row: Dict[str, Any], line_number: int, barcode_field: str, regex: str
     ) -> Tuple[str, str]:
-        """Extracts fields from a row of data (from the CSV file). Currently extracting the barcode
-        and coordinate (well position) using regex groups.
+        """Extracts fields from a row of data (from the CSV file). Currently extracting the barcode and coordinate (well
+        position) using regex groups.
+
+        We are using the re.ASCII flag so that \\w and \\W match ASCII values only.
 
         Arguments:
             row {Dict[str, Any]} -- row of data from CSV file
-            barcode_field {str} -- field indicating the plate barcode of interest, might also
-            include coordinate
+            barcode_field {str} -- field indicating the plate barcode of interest, might also include coordinate
             regex {str} -- regex pattern to use to extract the fields
 
         Returns:
             Tuple[str, str] -- the barcode and coordinate
         """
-        m = re.match(regex, row[barcode_field])
+        pattern = re.compile(regex, re.IGNORECASE | re.ASCII)
```

Comment:
What is the significance of defining the pattern with `re.IGNORECASE` and `re.ASCII`?

Reply:
I just double checked and the IGNORECASE flag isn't required, by defining ASCII it covers upper and lower case.

Reply:
What about for coordinates, e.g. `a01` vs. `A01`?

Reply:
Are you asking whether we should allow both?

Reply:
... yes, maybe. Specifically whether we want to allow lowercase coordinates, or whether to use `[A-Z]` and enforce uppercase like the previous implementation

Reply:
You're right, it's best to keep it as close to current implementation until they notice something and would like to allow lowercase. I'll update the regex and flags.